GameData/ROEngines/PartConfigs/RD180_BDB.cfg

```diff
@@ -0,0 +1,220 @@
+PART
+{
+	module = Part
+	name = ROE-RD180
+	author = CobaltWolf
+
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
+
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+	
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/AtlasV/bluedog_AtlasV_RD180
+	}
+
+	scale = 1.0
+	rescaleFactor = 1.6
+	node_stack_top = 0.0, 0.4122696, 0.0, 0.0, 1.0, 0.0, 2
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0
+
+	title = RD-180
+	manufacturer = NPO Energomash
+	description = The RD-180 is a two-chamber derivative of the four-chamber RD-170/171 and powers the first stage of the Atlas V. Diameter: [3.5 m].
+
+	tags = rd180, rd-180, 180, atlas-v, atlas, v, atlas-5, 5
+
+	engineType = RD180
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
+
+	TechRequired = stagedCombustion1998
+	cost = 2700
+	entryCost = 54000
```

Comment:
Can you make these parts [and your other PRs] use the tool in https://github.com/KSP-RO/ROEngines/blob/PatchingFixes/GameData/ROEngines/ROEEngineDefaults.cfg?